core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -17,16 +17,15 @@
 
 package org.apache.spark.status
 
-import java.io.File
-import java.util.{Arrays, List => JList}
+import java.util.{List => JList}
 
 import scala.collection.JavaConverters._
 
 import org.apache.spark.{JobExecutionStatus, SparkConf}
 import org.apache.spark.scheduler.SparkListener
 import org.apache.spark.status.api.v1
 import org.apache.spark.ui.scope._
-import org.apache.spark.util.{Distribution, Utils}
+import org.apache.spark.util.{Distribution}
```

Comment:
No '{}' for a single class, but I'd just not touch this file if you're not actually making changes to the code.

Reply:
restored original